### ISSUE TITLE
Partials improvements

### ISSUE THREAD

**Issue by pmbrull** [open-metadata/docs-v1#134]
- [x] partial files should support full path import, not just file name. This is required to work with multiple versions.
- [x] We should also support passing variables to the markdoc components (note the caveats https://markdoc.dev/docs/variables#caveats)

Examples for variables we need:

```
{% tile
    title="Ingest with the CLI"
    description="Run a one-time ingestion using the metadata CLI"
    link="/connectors/database/$connector/cli"
  / %}
```

```
{% image
src="/images/v1.1.0/connectors/$connectorPath/select-service.png"
alt="Select Service"
caption="Select your service from the list" /%}
```

**Comment by aniketkatkar97** [comment]
We should pass the whole string for using the variables in such cases.
For example
```
{% tile
    title="Ingest with the CLI"
    description="Run a one-time ingestion using the metadata CLI"
    link=$connectorLink
  / %}
```

and pass variable as 
```
{% partial file="fileName.md" variable={connectorLink: "/connectors/database/Athena/cli"} /%}
```

**Comment by aniketkatkar97** [comment]
fixed in #139
